packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -156,6 +155,9 @@ class MarkdownBuilder implements md.NodeVisitor {
   /// does not allow for intrinsic height measurements.
   final MarkdownListItemCrossAxisAlignment listItemCrossAxisAlignment;
 
+  /// Collection of custom block tags to be used building block widgets.
+  final List<String>? customBlockTags;
```

Comment:
Nullable collections are generally a red flag; what is the difference between empty and null? If there is one, it should be clearly documented; if there isn't it then it should be non-nullable with a default value of empty.